src/Dropdown/Dropdown.js

```diff
@@ -71,7 +72,7 @@ export default class Dropdown extends React.Component {
   }
 
   show = () => {
-    const { placement, boundariesElement } = this.props;
+    const { offset = '0, 10', placement, boundariesElement } = this.props;
```

Comment:
add `offset` to `propTypes` and `defaultProps`?

Reply:
Thanks @choochootrain, great idea. Updated.